gateway/single-cluster/regional-l7-ilb/README.md

```diff
@@ -0,0 +1,259 @@
+# Single-cluster Gateway with Regional L7 Internal Load Balancing
+
+This recipe provides a basic walk through for setting up Single-cluster Gateway with the `gke-l7-rilb` GatewayClass, provisioning an regional internal HTTP/S Load Balancer.
+
+To achieve this, we will:
+
+- Deploy v1 of a sample application into the GKE cluster named `gke-1`, as well as its ServiceExport.
```

Comment:
You are deploying 2 versions in this example right ? If Yes than the instructions should reflect this. 

Also you are not using ServiceExport aren't you ? You are only using a Kubernetes Service ? ServiceExport is a specific concept